src/main/webapp/app/exercises/shared/team/team-participation-table/team-participation-table.component.html

```diff
@@ -0,0 +1,134 @@
+<jhi-data-table [isLoading]="isLoading" [showPageSizeDropdownAndSearchField]="false" [allEntities]="participations">
+    <ng-template let-settings="settings" let-controls="controls">
+        <ngx-datatable
+            class="bootstrap"
+            [limit]="settings.limit"
+            [sortType]="settings.sortType"
+            [columnMode]="settings.columnMode"
+            [headerHeight]="settings.headerHeight"
+            [footerHeight]="0"
+            [rowHeight]="settings.rowHeight"
+            [rows]="settings.rows"
+            [rowClass]="rowClass(exercise)"
+            [scrollbarH]="settings.scrollbarH"
+        >
+            <ngx-datatable-column prop="exercise.title" [minWidth]="140" [width]="140" [frozenLeft]="true">
+                <ng-template ngx-datatable-header-template>
+                    <span class="datatable-header-cell-wrapper" (click)="controls.onSort('exercise.title')">
+                        <span class="datatable-header-cell-label bold sortable" jhiTranslate="artemisApp.exercise.exercise">
+                            Exercise
+                        </span>
+                        <fa-icon [icon]="controls.iconForSortPropField('exercise.title')"></fa-icon>
+                    </span>
+                </ng-template>
+                <ng-template ngx-datatable-cell-template let-value="value">
+                    {{ value }}
+                </ng-template>
+            </ngx-datatable-column>
+
+            <ngx-datatable-column prop="exercise.releaseDate" [minWidth]="125" [width]="125" [maxWidth]="130" [frozenLeft]="true">
+                <ng-template ngx-datatable-header-template>
+                    <span class="datatable-header-cell-wrapper" (click)="controls.onSort('exercise.releaseDate')">
+                        <span class="datatable-header-cell-label bold sortable" jhiTranslate="artemisApp.exercise.releaseDate">
+                            Release Date
+                        </span>
+                        <fa-icon [icon]="controls.iconForSortPropField('exercise.releaseDate')"></fa-icon>
+                    </span>
+                </ng-template>
+                <ng-template ngx-datatable-cell-template let-value="value" class="mr-5">
+                    {{ moment(value) | date: 'dd.MM.yyyy' }}
```

Comment:
Apart from this small bug with a undefined date, everything works as expected!

Good work!

Reply:
Thanks!

The case of no release date being set is handled now:

![OnPaste 20200428-133740](https://user-images.githubusercontent.com/7109225/80482962-75373800-8955-11ea-99e8-cfb17b4aa4c4.png)

Reply:
Nice!